lib/graphql/schema.ts

```diff
@@ -1171,6 +1171,7 @@ export type MutationCreateTransactionAssetArgs = {
   filetype: Scalars['String'];
   name: Scalars['String'];
   transactionId: Scalars['ID'];
+  uploadPlatform?: InputMaybe<Scalars['String']>;
```

Comment:
don't we have an enum for that?

Reply:
I have proper type when i do:
```
@Arg("uploadPlatform", (type) => RequestPlatform, { nullable: true })
    uploadPlatform?: RequestPlatform
```

Reply:
Adjusted BE here: https://github.com/kontist/backend/pull/12046